app/controller.py

```diff
@@ -188,3 +195,51 @@ def troll_katie():
         turn_light_off()
         sleep(0.13)
     return redirect(('/' + get_current_day_name()).lower())
+
+"""
+Made initially for testing sound.
+Use it now and then to check if everything working properly.
+"""
+@app.route('/sound')
+def yolo_def():
+    from time import sleep
```

Comment:
Any reason we're not including this at the top of the file with the other imports?  We don't want to import it every time we visit the `/sound` route